rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/util/FormKeyTextExtractionUtil.java

```diff
@@ -69,7 +69,7 @@ public static void destroyFormWidgetKeyToTextMap() {
         formWidgetKeyToTextMap = null;
     }
 
-    private static JSONObject getTranslatedForm(String formName) throws JSONException {
+    public static JSONObject getTranslatedForm(String formName) throws JSONException {
```

Comment:
This can be called using `Reflectionhelpers` and doesn't need to be made public.

Reply:
fixed.